app/theme/index.ts

```diff
@@ -0,0 +1,6 @@
+import { extendTheme } from '@chakra-ui/react';
+import colors from './colors';
+
+const theme = extendTheme({ colors });
```

Comment:
Should we set the default `colorTheme` prop for Buttons, etc. as well (i.e., so you don't have to remember to do it each time)?  See https://github.com/chakra-ui/chakra-ui/discussions/2840

Reply:
I agree, this would be really helpful

Reply:
We can do it in a follow-up, @Myrfion/@Ririio if one of you wants to file it.